src/main/java/net/coderbot/iris/uniforms/FogUniforms.java

```diff
@@ -18,6 +19,7 @@ public static void addFogUniforms(DynamicUniformHolder uniforms, FogMode fogMode
 		if (fogMode == FogMode.OFF) {
 			uniforms.uniform1f(UniformUpdateFrequency.ONCE, "fogDensity", () -> 0.0F);
 			uniforms.uniform1i(UniformUpdateFrequency.ONCE, "fogMode", () -> 0);
+			uniforms.uniform1i(UniformUpdateFrequency.ONCE, "fogShape", FogShape.CYLINDER::getIndex);
```

Comment:
I don't like the idea of not defining what this means ourselves & just passing through the vanilla enum, especially if the vanilla enum changes.